README.md

```diff
@@ -92,26 +92,74 @@ Two specific points to expand on:
 
 ### Setting up
 
-1. Follow the [Flutter installation guide](https://docs.flutter.dev/get-started/install)
-   for your platform of choice.
```

Comment:
I think this still belongs in the instructions, because it sets up other pieces like the IDE and the Android or iOS build stack. But it can go after setting up the pinned Flutter, so that it can use that version.